articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -85,21 +82,24 @@ Once the resources are created, add the following environment variables. These w
 Open *Program.cs* and update the file with the following code.
 
 ```csharp
+using Azure.Messaging.EventGrid;
 using Microsoft.Azure.ServiceBus;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions;
 using System;
-using System.Diagnostics;
-using System.Text;
-using System.Text.Json;
 using System.Threading.Tasks;
 
 namespace TestConsole
 {
     class Program
     {
-        private const string AppConfigurationConnectionStringEnvVarName = "AppConfigurationConnectionString"; // e.g. Endpoint=https://{store_name}.azconfig.io;Id={id};Secret={secret}
-        private const string ServiceBusConnectionStringEnvVarName = "ServiceBusConnectionString"; // e.g. Endpoint=sb://{service_bus_name}.servicebus.windows.net/;SharedAccessKeyName={key_name};SharedAccessKey={key}
+        private const string AppConfigurationConnectionStringEnvVarName = "AppConfigurationConnectionString";
+        // e.g. Endpoint=https://{store_name}.azconfig.io;Id={id};Secret={secret}
+        
+        private const string ServiceBusConnectionStringEnvVarName = "ServiceBusConnectionString";
+        // e.g. Endpoint=sb://{service_bus_name}.servicebus.windows.net/;SharedAccessKeyName={key_name};SharedAccessKey={key}
```

Comment:
I know this is not your code, but we shouldn't lead customers to hard code their connection strings. They should read from environment variables. Can you help to update the code to below (yes, remove the "e.g" too)? Please run the code and make sure it works.

```cs
private const string AppConfigurationConnectionStringEnvVarName = Environment.GetEnvironmentVariable("AppConfigurationConnectionString");
private const string ServiceBusConnectionStringEnvVarName = Environment.GetEnvironmentVariable("ServiceBusConnectionString");

```